pkg/okteto/auth.go

```diff
@@ -42,18 +38,68 @@ type Token struct {
 	MachineID string `json:"MachineID"`
 }
 
-// Auth authenticates in okteto with an OAuth code
-func Auth(ctx context.Context, code, url string) (*types.User, error) {
-	oktetoClient, err := NewOktetoClientFromUrl(url)
-	if err != nil {
-		return nil, err
+type authenticationErr struct {
+	E error
+}
+
+func (authenticationErr) Error() string {
+	return "authentication error, please try again"
+}
+func (e authenticationErr) Unwrap() error {
+	return e.E
+}
+
+func newAuthenticationErr(err error) authenticationErr {
```

Comment:
I don't think we need this function, do we? We could use `authenticationErr` directly ?